Makefile

```diff
@@ -50,10 +50,10 @@ ctags:
 	$(CTAGS) --python-kinds=-i -R sklearn
 
 doc: inplace
-	$(MAKE) -C doc html
+	PYTHONPATH=$$PWD:$$PYTHONPATH $(MAKE) -C doc html
 
 doc-noplot: inplace
-	$(MAKE) -C doc html-noplot
+	PYTHONPATH=$$PWD:$$PYTHONPATH $(MAKE) -C doc html-noplot
```

Comment:
Wouldn't it be better to add the current dir (if it's necessary) to the path in `doc/config.py` as [it's already done for `sphinxext` there](https://github.com/scikit-learn/scikit-learn/blob/master/doc/conf.py#L24)? Also, why is this necessary (i.e. doesn't Circile CI run without it)?

Reply:
I hadn't actually meant to add this to the PR... `git commit -a` fail. But I'm not sure about forcing one to use the repository: if they did a site-packages install before building docs, that wouldn't work. I'm pretty ambivalent, though.